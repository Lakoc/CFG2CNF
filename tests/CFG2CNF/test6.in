S,A,B
a,b
S
S->ab