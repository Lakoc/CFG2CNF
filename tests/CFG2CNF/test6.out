S,A,B,a',b'
a,b
S
S->a'b'
a'->a
b'->b